tools/clang/test/HLSLFileCheck/hlsl/objects/Texture/gatherCmp.hlsl

```diff
@@ -14,26 +14,26 @@ float4 main(float2 a : A) : SV_Target
   r += text1.GatherCmp(samp1, a, cmpVal, uint2(-3, 2), status); r += status;
 
   r += text1.GatherCmpAlpha(samp1, a, cmpVal);
-  r += text1.GatherCmpAlpha(samp1, a, cmpVal, uint2(-3,8));  
-  r += text1.GatherCmpAlpha(samp1, a, cmpVal, uint2(-3,8),status); r += status;    
+  r += text1.GatherCmpAlpha(samp1, a, cmpVal, uint2(-3,7));
+  r += text1.GatherCmpAlpha(samp1, a, cmpVal, uint2(-3,7),status); r += status;
   r += text1.GatherCmpAlpha(samp1, a, cmpVal, uint2(-3,8),uint2(-2,3), uint2(-3,8),uint2(-2,3));  
   r += text1.GatherCmpAlpha(samp1, a, cmpVal, uint2(-3,8),uint2(8,-3), uint2(8,-3), uint2(-3,2), status); r+=status;  
```

Comment:
Chose to leave the 4-offset variant with values outside the range even though I doubt that was the original intent. It verifies that these are still possible